src/main/java/com/aws/greengrass/mqttbridge/MQTTBridge.java

```diff
@@ -118,29 +120,37 @@ public void install() {
 
     @Override
     public void startup() {
+        String serverUri = Coerce.toString(config.findOrDefault(MQTTClient.DEFAULT_BROKER_URI,
```

Comment:
You'll need to subscribe to this topic so that if it changes in a deployment, the bridge will reconfigure itself. We could also setup bridge to just restart itself if any of these important fields change.

Reply:
Yeah, there's TODO for this in `MQTTClient`, didn't know if it would be better to split out into its own PR, but can add that here 👍